Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_ASM_Alert.yml

```diff
@@ -2658,43 +2810,40 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
-  "113":
-    id: "113"
-    taskid: 3660106c-2c7e-4a5a-8d77-13c38f87c6c4
+  "126":
+    id: "126"
+    taskid: ac937d90-ae1a-41e3-8aa4-69a469a5960d
     type: regular
     task:
-      id: 3660106c-2c7e-4a5a-8d77-13c38f87c6c4
+      id: ac937d90-ae1a-41e3-8aa4-69a469a5960d
       version: -1
-      name: Close alert (close now) with rule match
-      description: Close the current alert because auto-close was selected,
-      script: Builtin|||closeInvestigation
+      name: Set playbook stage grid field (analysis)
+      description: |-
+        Automation used to more easily populate a grid field.  This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well.  Instead of a value you can enter `TIMESTAMP` to get the current timestamp in ISO format.  Example of command:
```

Comment:
```suggestion
        Automation used to more easily populate a grid field. This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well. Instead of a value, you can enter `TIMESTAMP` to get the current timestamp in ISO format.  Example of the command:
```